grc/gui_qt/components/window.py

```diff
@@ -159,8 +159,14 @@ def __init__(self):
         self.tabWidget.tabCloseRequested.connect(
             lambda index: self.close_triggered(index)
         )
+        self.tabWidget.tabBarClicked.connect(
```

Comment:
I'm thinking maybe we should use the `currentChanged` signal instead of `tabBarClicked`, in case we want to be able to navigate tabs using Alt+numbers.